tests/smoke-tests/test-balances-consistency.ts

```diff
@@ -0,0 +1,241 @@
+import "@moonbeam-network/api-augment";
+import { ApiDecoration } from "@polkadot/api/types";
+import type { FrameSystemAccountInfo } from "@polkadot/types/lookup";
+import { expect } from "chai";
+import { printTokens } from "../util/logging";
+import { describeSmokeSuite } from "../util/setup-smoke-tests";
+const debug = require("debug")("smoke:balances");
+
+const wssUrl = process.env.WSS_URL || null;
+
+describeSmokeSuite(`Verify balances consistency`, { wssUrl }, (context) => {
+  const accounts: { [account: string]: FrameSystemAccountInfo } = {};
+
+  let atBlockNumber: number = 0;
+  let apiAt: ApiDecoration<"promise"> = null;
+
+  before("Retrieve all balances", async function () {
+    // It takes time to load all the accounts.
+    this.timeout(600000);
```

Comment:
:eyes:

Reply:
Well, we have 500k+ accounts on Moonriver